Fix various typos, rephrase some language (#164)
data-preservation-programs/singularity: docs/en/README.md

* a push mode where Singularity clients define how to send out deals - whether it's an one-off batch or a scheduled cronjob.
* a pull mode where Storage providers can ask for deal proposals themselves as long as it satisfy the policies set by Singularity clients.

### Push Mode

Singularity clients can set up a schedule with storage providers to send out a set number of deals for each interval time. They can also configure the maximum deals or deal size for each schedule as well as how many in-flight deals are allowed. Singularity handles deal renewal when a deal is slashed, expires or disappears on chain for any reason.

### Pull Mode

Singularity allows storage providers to query for a list of PieceCIDs that they can propose as deals to themselves as long as those storage providers are allowlisted in the deal making self service policy. This way, storage providers have ultimate flexibility to control the deal flow within their sealing pipelines.

### Wallet Management

Singularity allows clients to import a Filecoin wallet using private keys and associate multiple wallets with datasets. It regularly checks datacap of each wallet to load balance between different wallets.

> Singularity also supports remote signer so the client can keep their wallet private key known only to them.

Reviewer: ```suggestion
Singularity also supports remote signing so the client can keep their wallet private key outside of the system.
```

Author: Sounds good, thanks for the rephrasing as even my version was still a little awkward.
